addons/l10n_eu_service/__init__.py

```diff
@@ -3,3 +3,21 @@
 
 from . import wizard
 from . import models
+
+from odoo import api, SUPERUSER_ID
+
+def _post_install_hook_map_eu_taxes(cr, registry):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    eu_countries = env.ref('base.europe').country_ids
+    companies = env['res.company'].search([])
+    companies.filtered(lambda c: c.country_id in eu_countries).map_eu_taxes()
```

Comment:
Not sure, but a sudo() might be needed to be allowed to access everything and ignore the record rules; I don't know if the super user is already the one running this.

Reply:
Both the install and uninstall hooks will be removed. Unless we want to clear the config parameter